__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const seed = require("../db/seeds/seed.js");
+const app = require("../app.js");
+const request = require("supertest");
+const data = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/topics", () => {
+  test("200, respond with array of objects with property of slug & description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const topicsArray = body.topics;
+
+        expect(topicsArray).toHaveLength(3);
```

Comment:
This is great. If you had only done the tests in the forEach below, without also checking for the length of the returned array as you have above: we would run the risk of receiving an empty array, which would cause the block from 25 onwards to give us a false positive (if it checks 'each' element of an empty array, it's never going to get a negative response). Good job!